utils/linear_probes.py

```diff
@@ -0,0 +1,422 @@
+import os
+import pathlib
+os.environ["OMP_NUM_THREADS"] = "6" # export OMP_NUM_THREADS
+os.environ["OPENBLAS_NUM_THREADS"] = "6" # export OPENBLAS_NUM_THREADS
+os.environ["MKL_NUM_THREADS"] = "6" # export MKL_NUM_THREADS
+os.environ["VECLIB_MAXIMUM_THREADS"] = "6" # export VECLIB_MAXIMUM_THREADS
+os.environ["NUMEXPR_NUM_THREADS"] = "6" # export NUMEXPR_NUM_THREADS
```

Comment:
Does this really help?

Reply:
I mean all these envs

Reply:
Yep -- oftentimes logistic regression from sklearn would take up all CPU cores and make my workstation hang. Probably not all env vars are needed but I don't want to figure out which one is specifically responsible for workstation hanging

Reply:
Ok